classes/02class/exercises/c04-iam01/jayanath/README.md

```diff
@@ -0,0 +1,83 @@
+# c04-iam01
+
+## Command Execution Output
```

Comment:
Hi @jayanath , can you move your answer to a file `cli_commands.txt` as indicated in the instructions?

Reply:
Thanks @denstorti. Moved the file as advised. Ready to review again.

Reply:
Thanks @jayanath . You need to keep the README.md file as it has a reference to the exercise version you answered in case we improve the exercise in the future.

Check out this example: https://github.com/devopsacademyau/academy/tree/master/classes/02class/exercises/c04-iam01/starkmatt

Reply:
Added the README.md as well. Thanks @denstorti  👍